docs/synchronizations-faq.md

```diff
@@ -0,0 +1,43 @@
+# Synchronizations FAQ
+##
+
+In Private Packagist synchronizations is how you keep packages, teams, their members, and access permissions in sync with your version control system like GitHub, GitLab or Bitbucket.
+
+You can find below answers for frequently-asked questions concerning synchronizations.
+
+## What happens if you promote a synchronization to a primary synchronization?
+
+Only with a primary synchronization that admins and owners teams will be added to Private Packagist.
+
+The following are some things to consider when promoting a synchronization to a primary synchronization:
+- Owners and Admins teams in Private Packagist will be the same ones from the primary synchronization.
+- In case you change the integration (e.g. From GitHub to Bitbucket), you'll have to go to your profile page and connect your Private Packagist account with the new integration. Otherwise, you'll lose access to your account.
+- You will not be able to delete the primary synchronization.
```

Comment:
The first and last entry here are not necessary. Remove the list and instead build out the second point into a section that explains what happens if you promote a new synchronization to primary: new admins and owners will be added to the teams, this means people not in the new team will lose access and explain what needs to be done when the new primary sync is on a different platform/integration. Users will not lose access to their account. They will lose access to the organization. On the profile page they are not called integrations but services.

Reply:
ok I removed the two entries and rewrote this.